src/test/libcephfs/TestClient.h

```diff
@@ -0,0 +1,12 @@
+#ifndef CEPH_TESTCLIENT_H
+#define CEPH_TESTCLIENT_H
+
+#include "client/UserPerm.h"
+#include "client/Client.h";
+
+class TestClient: public StandaloneClient {
```

Comment:
You need to define a destructor, even if it's empty.

Reply:
I think this can go in `src/test/client/` but the name needs to be different (there already is a `TestClient` class there!).

Reply:
Done.